.changeset/itchy-mayflies-allow.md

```diff
@@ -0,0 +1,11 @@
+---
+'braid-design-system': patch
+---
+
+---
+updated:
+  - Text
+  - Heading
+---
+
+**Text, Heading:** Fix `maxLines` cropping of decending characters
```

Comment:
We encourage adding a bit of context about the fix after the summary for those wanting to know more, something like this maybe:

```suggestion
**Text, Heading:** Fix `maxLines` cropping of decending characters

Fixes a bug when using -webkit-box, where the descender on the last line of text could be cropped based on the combination of line height and font size.
```